data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java

```diff
@@ -210,7 +189,7 @@ public void populateBeans(UniqueList<Table> finalInTableNodes, UniqueList<Table>
 					System.out.println("source and destination are not same columns = " + relation);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)